### ISSUE TITLE
Wordcloud is not working when words get set dynamically

### ISSUE THREAD

**Issue by langersteff** [robinboehm/angular-d3-directives#8]
I'm pulling words via an API and I want to show them with the wordcloud directive. 
But, if the words array gets set later or updated, the wordcloud doesn't work.

```
setTimeout(function(){
  $scope.words = ["foo","bar"];
  $scope.$apply();
}, 2000);
```

I used the code above for testing the "delay" caused by pulling the words.

**Comment by robinboehm** [comment]
Yes, you need to Impl a simple watch for this that triggers the recalculation and rearrangement.
Are you able to implement this patch and do a PR?
